### PR TITLE
Working on Fates visualizations

### PR DESCRIPTION
**Pull request by diggyk**
Finally got a working visualization of the fates.  Needs some work in the future to make it more explorable using pan and zoom.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ gulp.task('build:js', function() {
         //.pipe(sort())
         .pipe(concat('app.js'))
         .pipe(gulp.dest((BUILD_DEST + 'js')))
-        .pipe(uglify())
+        //.pipe(uglify())
```
Comment (gmjosack) [inline gulpfile.js]: Is this intentionally commented out? If so the following two lines don't make much sense either

File: hermes/webapp/src/index.html [hunk 2]
```diff
@@ -12,10 +12,11 @@
         <div id="header">Hermes</div>
         <div style="padding: 10px" ng-view></div>
     </div>
-
     <script src="/vendor/angular/angular.js"></script>
     <script src="/vendor/angular-route/angular-route.js"></script>
     <script src="/vendor/angular-animate/angular-animate.js"></script>
+    <!--<script src="/vendor/sigma.js/sigma.min.js"></script>-->
```
Comment (gmjosack) [inline hermes/webapp/src/index.html]: Could be missing it but is this intentional? I don't see sigma included anywhere else.

File: bower.json [hunk 3]
```diff
@@ -18,6 +18,8 @@
     "angular": "~1.4.5",
     "angular-animate": "~1.4.5",
     "angular-route": "~1.4.5",
-    "bootstrap": "~3.3.5"
+    "bootstrap": "~3.3.5",
+    "raphael": "raphael.js#~2.1.4",
+    "sigma.js": "*"
```
Comment (gmjosack) [inline bower.json]: We should lock down on a specific version here instead of just *

File: gulpfile.js (hunk 1 above)
Comment (diggyk) [inline gulpfile.js]: I had a suspicion it was borking some of my code but it seems to be working now so I'll put it back in.

File: hermes/webapp/src/index.html (hunk 2 above)
Comment (diggyk) [inline hermes/webapp/src/index.html]: Ah yes.  I implemented all of this in Sigma but I couldn't get it work like I wanted to so I ripped Sigma out and replaced it but missed this line.

File: bower.json (hunk 3 above)
Comment (diggyk) [inline bower.json]: Weird!  I just did `bower install sigma.js --save`.  Anyway, I'm not using sigma now so I'll just remove this.

**Pull request merged**